3drepo/3drepo.io
# ISSUE #3081 - V5 Login Screen

This fixes #3081 

#### Description
- Adds login screen for v5
- If you are in the v5 part of the app and authenticated you get redirected to /v5/login
- If you try to navigate to /v5/login and you are authenticated you will get redirected to /v5/dashboard
- Saga testing for login

#### Test cases
- Log in
- Log out
- Log in with incorrect credentials. Observe the error message.
- When logged in attempt to navigate to /v5/login. You will be redirected to dashboard.
- When logged out attempt to navigate to another part of the app (e.g. v5/dashboard) you will be redirected to the login screen.
- Log out and try the v4 login. This should be unaffected.
- Login too many times with correct username but incorrect password. Observe the account lockout message.

#### Notes
- The SVG background is subject to change
- The links currently point to the v4 URLs. The release notes currently links to a dead link as awaiting the location for that.
- The input labels are currently in all caps as this will change when the control changes in ISSUE #[3084](https://github.com/3drepo/3drepo.io/issues/3084)
- The profile fetching logic during authentication is missing as this requires ISSUE #3049. All this currently means is that your profile image is missing in the top right.
